Classes/Core/ConsoleBootstrap.php

```diff
@@ -96,6 +103,14 @@ public function initialize(\Composer\Autoload\ClassLoader $classLoader)
     }
 
     /**
+     * @return ClassLoader
+     */
+    public function getClassLoader()
```

Comment:
The class loader is already available as "earlyInstance" no need to add a new property and public getter for that